starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -865,8 +868,16 @@ sub check_dither {
 
     # Check for standard dither
     if ($guide_dither->{state} eq 'ENAB'){
-        if ((not standard_dither($guide_dither)) or not standard_dither($acq_dither)){
-            push @{$self->{yellow_warn}}, "Non-standard dither\n";
+	if ((not standard_dither($guide_dither)) or not standard_dither($acq_dither)){
```

Comment:
I think this is conflating guide dither and monitor commanding in a way that is not quite right. Since guide dither is the state at +8 minutes into the observation and monitor commanding ends at +5 minutes, they are basically unrelated and so should be treated separately.

Roughly speaking I think the logic should be about:
```
if catalog has MON:
  if acq_dither amplitude is not 0.0 in either component:
    red warning (this would be a benign commanding error if dither is disabled)
  else:
    FYI message (if it were me I would not emit this, but I know you like positive confirmation)

  if not standard_dither($guide_dither):
    yellow_warn "Non-standard dither"

else:
  if ((not standard_dither($guide_dither)) or not standard_dither($acq_dither)){
            push @{$self->{yellow_warn}}, "Non-standard dither\n";

Reply:
If an observation has non-standard guide dither, then that incorrectly blocks testing of the monitor window dither amplitude and you will always get a "Non-standard dither" warning regardless of whether the "Monitor window dither transition" is correct.

Reply:
The current test is also relying on the historical circumstance of 0x0 dither being "non-standard" to work [1]. I've always disliked "standard" vs "non-standard" dither and this criteria should not be a gating item for entering the logic to test whether the dither is 0.0 during the monitor sequence.

[1] There is no real reason we could not have decided long ago that 0x0" dither is a standard configuration and put that in the arbitrary list.

Reply:
> where you want to go with the 0x0 thoughts

Just that the check here for correct MON dither amplitude should not call `standard_dither()` anywhere. That happens if you follow my proposed flow.

Reply:
One thing that just occurred to me is that the non-standard dither check is being gated by the guide dither state being `ENAB`, but if for some reason dither is disabled then that should be a caution warning as well. I.e. we want to know if an observation is being scheduled with dither disabled. This warning should be emitted even if the OR says dither should be disabled since that would most likely be a mistake.

Since maintaining dither is turning into a spacecraft safety issue we need to be extra careful now.

Reply:
Sure.  If we want to add dither-better-be-on checks that seems fine.  Seems like they would not be as needed in sparkles, though also seems like proseco and sparkles shouldn't  let you have dynamic background bonus stars if dither disabled or too small (though that is in the weeds).